src/xxd/xxd.c

```diff
@@ -498,6 +517,11 @@ main(int argc, char *argv[])
   char *pp;
   char *varname = NULL;
   int addrlen = 9;
+  int color=0;
```

Comment:
Missing space around `=`, ditto for several other places in the newly-added code.